src/placeholder/styles/css/style.css

```diff
@@ -1,135 +1,37 @@
-@media (max-width: 767.98px) {
-  h1 {
-    font-weight: 80;
-    font-size: 53px;
-  }
-  .logo {
-    position: absolute;
-    height: 70px;
-    width: 70px;
-    top: 5px;
-    right: 5px;
-  }
-  .social-link {
-    height: 80px;
-    width: 80px;
-    margin: 5px;
-  }
-  .coming-back h2 {
-    font-size: 30px;
-    font-weight: 500;
-  }
+/* Error: Undefined mixin.
```

Comment:
That is a really bad sign - you need to fix the placeholder app and run the build again. We cannot put this code on the server - placeholder cannot be broken.

Reply:
As before, the "install npm" command helped.